src/js/components/uploadDetachedFiles/uploadDetachedFileMeta.jsx

```diff
@@ -0,0 +1,458 @@
+/**
+* UploadDetachedFileMeta.jsx
```

Comment:
This component very large - can we break it down into smaller components to handle individual sections of the page?